packages/utils/src/__tests__/normalizeErrors.js

```diff
@@ -0,0 +1,156 @@
+import normalizeErrors from "../normalizeErrors";
+
+const errors = [
+  {
+    message: "Root message",
+    locations: {},
+    path: [],
+    extensions: {
+      code: "UNPROCESSABLE_ENTITY",
+      exception: {
+        inputErrors: [
+          {
+            message: "Foo {bar} baz!",
+            options: { bar: "hello" },
+            path: ["settlement"]
+          }
+        ]
+      }
+    }
+  }
+];
+
+const complicatedErrors = [
+  {
+    message: "Root message",
+    locations: {},
+    path: [],
+    extensions: {
+      code: "UNPROCESSABLE_ENTITY",
+      exception: {
+        inputErrors: [
+          {
+            message: "Foo {bar} baz!",
+            options: { bar: "hello" },
+            path: ["UNPROCESSABLE_ENTITY", "address", "settlement"]
```

Comment:
I assume that such path values were handy for debugging, but I'm sure that they will be confusing for people who don't have much experience with our code

```suggestion
            path: ["address", "settlement"]
```